src/uct/api/v2/uct_v2.h

```diff
@@ -506,6 +549,25 @@ ucs_status_t uct_md_mkey_pack_v2(uct_md_h md, uct_mem_h memh,
                                  void *rkey_buffer);
 
 
+/**
+ * @ingroup UCT_MD
+ *
+ * @brief Locally attach to a remote memory.
```

Comment:
Can you please add full function description in addition to brief.

Reply:
done